apps/1d-stencil/1d-cuda.cpp

```diff
@@ -14,17 +16,17 @@ struct args_params_t : public argparse::Args {
   bool& k = kwarg("k", "Heat transfer coefficient").set_default(0.5);
   double& dt = kwarg("dt", "Timestep unit (default: 1.0[s])").set_default(1.0);
   double& dx = kwarg("dx", "Local x dimension").set_default(1.0);
-  bool& no_header =
-      kwarg("no-header", "Do not print csv header row (default: false)")
-          .set_default(false);
   bool& help = flag("h, help", "print help");
   bool& time = kwarg("t, time", "print time").set_default(true);
 };
 
 using Real_t = double;
+
```

Comment:
If it is being used in all 1d-xx.cpp files then this should go into a common header file.